server/TelegramServerConfig.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef TELEGRAMSERVER_CONFIG_HPP
```

Comment:
Please add license to the header and maybe cpp file. Use FunctionStreamOperators [header](https://github.com/Kaffeine/telegram-qt/blob/18e931673d669fb410160c85248ff9aecee65941/server/FunctionStreamOperators.hpp) and [source](https://github.com/Kaffeine/telegram-qt/blob/18e931673d669fb410160c85248ff9aecee65941/server/FunctionStreamOperators.cpp) files as reference.

Of course you have to use your name, not mine :rofl:.